src/dmd/dsymbol.d

```diff
@@ -991,6 +991,19 @@ extern (C++) class Dsymbol : RootObject
         return false;
     }
 
+    /****************************************
+    * extern(C++) needs to know this to mangle properly
+    */
+    final bool isPrimaryDtor()
```

Comment:
I would be nice with a description of what a primary destructor is.

Reply:
I could... but this is the only PR that has passed CI after many hours.
starting over feels... disappointing :/

Reply:
It is the outer/main destructor. Aggregates gather a list of destructors, but only one of them is the most inclusive destructor, ie `__xdtor`.